src/coverlet.collector/DataCollection/CoverletCoverageCollector.cs

```diff
@@ -130,12 +130,11 @@ private void OnSessionEnd(object sender, SessionEndEventArgs e)
                 _eqtTrace.Verbose("{0}: SessionEnd received", CoverletConstants.DataCollectorName);
 
                 // Get coverage reports
-                string coverageReport = _coverageManager?.GetCoverageReport();
+                var coverageReports = _coverageManager?.GetCoverageReports();
```

Comment:
`var` => `IEnumerable<Tuple<string, string>>` explicit if not clear from right part.

Reply:
Done